tests/page-objects/send-form.page.ts

```diff
@@ -95,4 +120,77 @@ export class SendPage {
     await this.page.waitForSelector('[data-asset="stx"]');
     await this.page.click('[data-asset="stx"]');
   }
+
+  async clickSTXTokenOption() {
```

Comment:
I search for all these but I don't see any tests using them? Were they lost rebasing?